### DISCUSSION TITLE
Closing the GUI stop the rules...

### DISCUSSION THREAD

**Discussion by BobSquarePants** [evilsocket/opensnitch#896] [Q&A] [answered]
Hi,

I'm testing the 1.6.0rc5

After closing the GUI by a right click on the OpenSnitch bar? icon
![2023-04-01-085451_](https://user-images.githubusercontent.com/23012674/229270771-52ab095b-d435-4171-af15-b268fb49bebb.png)

It seem that the opensnitch daemon let out all traffic.. it that the expected behavior ?

When I run this command after having closing the GUI `ps -AF | grep opensn`

```shell
/usr/bin/opensnitchd -rules-path /etc/opensnitchd/rules
```

**Comment by BobSquarePants** [comment]
I did more tests

so where no rule is specified and the GUI is closed then the packet are send.
but if a deny rule is in place and the gui is closed then the packet is blocked.

So it's seem that default when the GUI is closed it's the let the packet go trough if there is no rule about it.

Weird in the setting it setup otherwise ->
![default](https://user-images.githubusercontent.com/23012674/229272843-84f43787-b68e-44ba-8d09-c832b02dfc5f.png)

**Comment by BobSquarePants** [comment]
When I change **Default action when the GUI is disconnected** to allow, then apply then set back to deny.
It solve the problem (and it keep that way after reboot too)

so most probably the problem occur at install time.

**Comment by gustavo-iniguez-goya** [comment, marked as answer]
I've modified these options. I hope it'll be more clear now. I've described how it'll behave here: https://github.com/evilsocket/opensnitch/commit/74b6bc2c2924d6d6d421b3f332a98dee33bd1207
